src/api/v1.rs

```diff
@@ -286,10 +273,7 @@ fn today_code(
 /// Returns the current period type.
 #[openapi]
 #[get("/today/now?<timestamp>")]
-fn today_now(
-	schedule: State<Arc<RwLock<Schedule>>>,
-	timestamp: Option<i64>,
-) -> Json<Vec<Period>> {
+fn today_now(schedule: State<Arc<RwLock<Schedule>>>, timestamp: Option<i64>) -> Json<Vec<Period>> {
```

Comment:
Which Rust extension are you using? `matklad.rust-analyzer` is the one listed in the repo's recommended extensions file, and afaik this isn't it's default behavior.

Reply:
I was using `rust-lang.rust` but I just switched to analyzer.